packages/management/tanzu-auth/package.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: tanzu-auth.tanzu.vmware.com.1.6.0
+  namespace: tanzu-auth
+spec:
+  refName: tanzu-auth.tanzu.vmware.com
+  version: 1.6.0
```

Comment:
question: are you insinuating that we version the package alongside the TKG version? or is this just to get things working, per your note in slack

Reply:
Just to get things working!  Though I chose `1.6.0` because it seemed reasonable to match TKG, I am not strongly asserting we should do this.